stdlib/public/core/Codable.swift.gyb

```diff
@@ -1767,6 +1767,40 @@ extension Array : Decodable where Element : Decodable {
   }
 }
 
+extension ContiguousArray : Encodable where Element : Encodable {
```

Comment:
Shouldn't there be some tests for the `ContiguousArray` extensions?

Reply:
Yes, there should. This is being discussed in the comments to 
https://github.com/apple/swift/pull/20715. I'll update the PR once that's done.